tracim/development.ini.base

```diff
@@ -235,6 +237,16 @@ email.reply.use_txt_parsing = true
 # it's just an empty file use to prevent concurrent access to imap unseen mail
 email.reply.lockfile_path = %(here)s/email_fetcher.lock
 
+## Jitsi_meet_integration
+jitsi_meet.activated = False
+jitsi_meet.domain = your_jitsi_meet_instance
+jitsi_meet.use_token = False
+jitsi_meet.token_generator = local
+jitsi_meet.token_generator.local.app_id = tracim
```

Comment:
it's important to explain each parameter and its origin (is it a parameter required by jitsi meet ? if so, what value should be put ? add an example)